pkg/controller/nodelifecycle/metrics.go

```diff
@@ -79,6 +79,25 @@ var (
 		},
 		[]string{"zone"},
 	)
+
+	updateNodeHealthDuration = metrics.NewHistogram(
+		&metrics.HistogramOpts{
+			Subsystem:      nodeControllerSubsystem,
+			Name:           "update_node_health_duration_seconds",
+			Help:           "Duration in seconds for NodeController to update the health of a single node.",
+			Buckets:        metrics.ExponentialBuckets(0.001, 2, 16), // 1ms -> ~30s
```

Comment:
I'm not sure we need that large granularity. Having metrics.ExponentialBuckets(0.001, 4, 10) is probably good enough.